MeesGame/Gameplay/GameObjects/Player.cs

```diff
@@ -123,7 +128,9 @@ private set
 
         public bool HasKey()
```

Comment:
Make this function a general function that can lookup any item, and call it with the correct parameter